File: api_CBLA/src/main/java/com/daw/persistence/entities/Temporada.java
package com.daw.persistence.entities;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "Temporada")
@Getter
@Setter
@NoArgsConstructor
public class Temporada {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer id;
	
	@Column(length = 4)
	private Integer anio;
	
	@Column(length = 50)
	private String categoria;
}

File: front_BLA/src/app/models/temporada.ts
export class Temporada {
    private _id: number;
    private _anio: number;
    private _categoria: string;

    constructor(id: number, anio: number, categoria: string){
        this._id = id;
        this._anio = anio;
        this._categoria = categoria;
    }

    public get id(): number {   return this._id;}
    public set id(value: number) { this._id = value; }
    public get anio(): number { return this._anio; }
    public set anio(value: number) { this._anio = value; }
    public get categoria(): string { return this._categoria; }
    public set categoria(value: string) { this._categoria = value; }
}